lib/osf-components/addon/components/button/template.hbs

```diff
@@ -0,0 +1,11 @@
+{{#if (eq @layout 'fake-link')}}
+    {{!-- template-lint-disable link-href-attributes  --}}
```

Comment:
href gets passed in from caller

Reply:
hm, would it be cleaner to use a `<button>` with `display: inline`?

i suggested `<a role='button'>` because it seemed simpler but it sure complicates this would-be-3-line template

Reply:
Unless we have a really good reason for using an `a` tag, I'd be in favor of keeping it simple and sticking with a `button` element.

Reply:
sounds good 👍